lib/mix/tasks/group_uploader.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Mix.Tasks.GroupUploader do
+  use Mix.Task
+
+  alias Cadet.{Accounts, Course}
+
+  @moduledoc """
+  Mix Task to upload the tutorial groups and it's
+  participants
+  """
+
+  def run([groups_arg | [avengers_arg | _]]) do
+    Mix.Task.run("app.start")
+    # Removing unneccesary headers
+    groups =
+      groups_arg
+      |> Xlsxir.multi_extract()
+      |> Keyword.get(:ok)
+      |> Xlsxir.get_list()
+      |> Enum.drop(3)
+
+    avengers =
+      avengers_arg
+      |> Xlsxir.multi_extract()
+      |> Keyword.get(:ok)
+      |> Xlsxir.get_list()
+      |> Enum.drop(1)
+
+    upload_to_database(groups, avengers, nil)
+  end
+
+  def run(_), do: {:error, "Invalid arguments"}
```

Comment:
It's a mix task, people usually run it from CLI, so it's better to give something more tangible like `Logger.error/1`